definitions/npm/lodash_v4.x.x/flow_v0.63.x-/lodash_v4.x.x.js

```diff
@@ -3130,13 +3128,10 @@ declare module "lodash/fp" {
     defaultTo<T1: number, T2>(defaultValue: T2, value: T1): T1 | T2;
     defaultTo<T1: void | null, T2>(defaultValue: T2): (value: T1) => T2;
     defaultTo<T1: void | null, T2>(defaultValue: T2, value: T1): T2;
-    flow: $ComposeReverse;
-    flow(funcs: Array<Function>): Function;
-    pipe: $ComposeReverse;
-    pipe(funcs: Array<Function>): Function;
-    flowRight: $Compose;
-    flowRight(funcs: Array<Function>): Function;
-    compose: $Compose;
+    flow: ($ComposeReverse & (funcs: Array<Function>) => Function);
+    pipe: ($ComposeReverse & (funcs: Array<Function>) => Function);
+    flowRight: ($Compose & (funcs: Array<Function>) => Function);
+    compose: ($Compose & (funcs: Array<Function>) => Function);
```

Comment:
Funny fact: Flow doesn't have such function as compose in 4.x.x version, it's flowRight now :)

Reply:
Yeah, I wasn't sure about that either, I just fixed up what was there. So remove it altogether?

Reply:
Actually, for the `lodash/fp`, (4.x.x) `compose` is an alias to `flowRight`, so it is there.